src/sb_gazebo/robots/rover_2018.xacro

```diff
@@ -0,0 +1,287 @@
+            rpy="0 0 0"
+            />
+    </joint>
+    <box_link 
+        name="chassis"
+        x_pos="0"
+        y_pos="0"
+        z_pos="0"
+        x_size="${chassis_x_size}"
+        y_size="${chassis_y_size}"
+        z_size="${chassis_z_size}"
+        roll="0" 
+        pitch="0" 
+        yaw="0"
+        mass="10"
+        material_name="grey"
+        />
+
+    <xacro:macro name="primary_rocker" params="lr_prefix y_reflect">
+        <!-- TODO: Make this revolute model with a spring to resist rotation (but allow it) instead of just being fixed like it currently is -->
+        <joint name="${lr_prefix}_primary_rocker_to_chassis" type="revolute">
+            <parent link="chassis"/>
+            <child link="${lr_prefix}_primary_rocker"/>
+            <origin xyz="
+                ${primary_rocker_x_offset} 
+                ${primary_rocker_y_offset*y_reflect} 
+                ${primary_rocker_z_offset}
+                " 
+                rpy="
+                0
+                0
+                0
+                " 
+                />
+            <axis xyz="0 1 0" rpy="0 0 0" />
+            <limit effort="100" velocity="100"/>
+            <joint_properties damping="0.0" friction="0.0"/>
+            <limit lower="-20" upper="20" effort="100" velocity="100"/>
+        </joint>
+        <cylinder_link 
+            name="${lr_prefix}_primary_rocker"
+            x_pos="0" 
+            y_pos="0" 
+            z_pos="0" 
+            radius="${primary_rocker_radius}"
+            length="${primary_rocker_length}"
+            roll="${pi/2}" 
+            pitch="${pi/2}" 
+            yaw="0"
+            mass="1"
+            material_name="grey"
+            />
+    </xacro:macro>
+    <primary_rocker lr_prefix="L" y_reflect="1"/>
+    <primary_rocker lr_prefix="R" y_reflect="-1"/>
+
+    <xacro:macro name="primary_rocker_arm" params="lr_prefix fb_prefix x_reflect y_reflect">
+        <joint name="${lr_prefix}_${fb_prefix}_primary_rocker_to_primary_rocker_arm" type="fixed">
+            <parent link="${lr_prefix}_primary_rocker"/>
+            <child link="${lr_prefix}_${fb_prefix}_primary_rocker_arm"/>
+            <origin xyz="
+                ${primary_rocker_arm_x_offset*x_reflect}
+                ${primary_rocker_arm_y_offset*y_reflect}
+                ${primary_rocker_arm_z_offset}
+                " 
+                rpy="0 0 0"
+            />
+        </joint>
+        <cylinder_link 
+            name="${lr_prefix}_${fb_prefix}_primary_rocker_arm"
+            x_pos="0"
+            y_pos="0"
+            z_pos="0"
+            radius="${primary_rocker_arm_radius}"
+            length="${primary_rocker_arm_length}"
+            roll="0" 
+            pitch="${primary_rocker_arm_pitch*x_reflect}" 
+            yaw="0"
+            mass="1"
+            material_name="grey"
+            />
+    </xacro:macro>
+    <primary_rocker_arm lr_prefix="L" fb_prefix="FRONT" x_reflect="1" y_reflect="1"/>
```

Comment:
The above comment could also be addressed by expanding L to LEFT

Reply:
Changed `L/R` to `left/right`